package.json

```diff
@@ -118,7 +121,9 @@
     "webpack": "node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js",
     "test": "npm run lint && npm run jest",
     "test-ci": "npm run lint && npm run jest:update",
-    "test:watch": "npm run jest -- --watch"
+    "test:watch": "npm run jest -- --watch",
```

Comment:
To run cypress in the CI server, check this out https://docs.cypress.io/guides/guides/continuous-integration.html#Xvfb